VirtualFCS/Electrochemical/Hydrogen/FuelCell.mo

```diff
@@ -1,40 +1,41 @@
 within VirtualFCS.Electrochemical.Hydrogen;
 
 model FuelCell "Model for a single PEM fuel cell"
+  import SI = Modelica.Units.SI;
   //*** DEFINE REPLACEABLE PACKAGES ***//
   // Medium models
   replaceable package Cathode_Medium = Modelica.Media.Air.MoistAir;
   replaceable package Anode_Medium = Modelica.Media.IdealGases.SingleGases.H2;
   replaceable package Coolant_Medium = Modelica.Media.Water.ConstantPropertyLiquidWater;
   //*** DECLARE PARAMETERS ***//
   // Physical parameters
-  parameter Real mass(unit = "kg") = 1 "Mass of the cell";
-  parameter Real volume(unit = "m3") = 0.001 "Volume of the cell";
+  parameter SI.Mass mass = 1 "Mass of the cell";
+  parameter SI.Volume volume = 0.001 "Volume of the cell";
   // Thermal parameters
-  parameter Real heatCapacity(unit = "J/(kg.K)") = 800 "Specific Heat Capacity";
+  parameter SI.SpecificHeatCapacity heatCapacity = 800 "Specific Heat Capacity";
   // Stack design parameters
   parameter Real N_cell(unit = "1") = 1 "Number of Cells";
-  parameter Real A_cell(unit = "m2") = 0.0237 "Active Area of the Cell";
+  parameter SI.Area A_cell = 0.0237 "Active Area of the Cell";
   // Electrochemical parameters
-  parameter Real i_0(unit = "A") = 0.0002 "Exchange Current";
-  parameter Real i_L(unit = "A") = 520 "Maximum Current Limit";
-  parameter Real i_x(unit = "A") = 0.001 "Cross-over Current";
+  parameter SI.Current i_0 = 0.0002 "Exchange Current";
+  parameter SI.Current i_L = 520 "Maximum Current Limit";
+  parameter SI.Current i_x = 0.001 "Cross-over Current";
   parameter Real b_1(unit = "V/dec") = 0.025 "Tafel Slope";
   parameter Real b_2(unit = "V/dec") = 0.25 "Transport Limitation Factor";
-  parameter Real R_0(unit = "Ohm") = 0.02 "Ohmic Resistance";
-  parameter Real R_1(unit = "Ohm") = 0.01 "Charge Transfer Resistance";
-  parameter Real C_1(unit = "F") = 3e-3 "Double Layer Capacitance";
+  parameter SI.Resistance R_0 = 0.02 "Ohmic Resistance";
+  parameter SI.Resistance R_1 = 0.01 "Charge Transfer Resistance";
+  parameter SI.Capacitance C_1 = 3e-3 "Double Layer Capacitance";
   //*** DECLARE VARIABLES ***//
   // Physical constants
-  Real R = 8.314;
-  Real F = 96485;
+  import R = Modelica.Constants.R;
+  import F = Modelica.Constants.F;
   // Fuel cell variables
-  Real V_cell;
-  Real j;
-  Real P_th;
-  Real p_H2(min = 0);
-  Real p_O2(min = 0);
-  Real p_0 = 100000;
+  SI.Voltage V_cell;
+  SI.CurrentDensity j;
+  SI.Power P_th;
+  SI.Pressure p_H2(min = 0);
+  SI.Pressure p_O2(min = 0);
+  SI.Pressure p_0 = 100000;
```

Comment:
I suggest replacing `SI.Pressure` with `SI.AbsolutePressure` all places where absolute pressure is actually used (`min=0` will then be declared in the type and hence doesn't need to be declared for each variable), or with `SI.PressureDifference` any other places (assuming atmospheric pressure as reference pressure unless otherwise noted).